File: app/src/main/java/com/rdev/tryp/model/LoginResponse.java
package com.rdev.tryp.model;

import com.google.gson.annotations.SerializedName;

public class LoginResponse {

    @SerializedName("data")
    private Data data;

    public void setData(Data data) {
        this.data = data;
    }

    public Data getData() {
        return data;
    }

    @Override
    public String toString() {
        return
                "LoginResponse{" +
                        "data = '" + data + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/rdev/tryp/model/LoginResponse.kt
package com.rdev.tryp.model

import com.google.gson.annotations.SerializedName


class LoginResponse {

    @SerializedName("data")
    var data: Data? = null

    override fun toString(): String {
        return "LoginResponse{" +
                "data = '" + data + '\''.toString() +
                "}"
    }

}